demo/configuration_section.js

```diff
@@ -55,6 +55,25 @@ shakaDemo.setupConfiguration_ = function() {
 
 
 /** @private */
+shakaDemo.updateRobustnessSuggestions_ = function() {
+  var drmInfo = shakaDemo.player_.drmInfo();
```

Comment:
I appreciate the idea, but this won't work as you would expect.

drmInfo() returns info about the currently-loaded stream.  Before you load anything, it returns null.

The robustness settings, however, have no immediate effect.  They effect the next content you load.

So using the key system of the current content to decide what values to suggest for the next content is not going to work the way it should in all cases.

An alternative would be to detect what key systems are supported by the browser, and suggest all known robustness settings for all supported key systems.  (Currently, the only key system with well-known values is Widevine, but we hope to see that change.)